### PR TITLE
Translation Dandelion_en for Dandelion_pt-br

### PR DESCRIPTION
**Pull request by nelsonrenan**

### CODE REVIEW

**Comment by lh1008** [comment]
Hey @nelsonrenan thank you for the translation. Pinging @botoserelepe @netrik182 for review.

**Review by botoserelepe — CHANGES_REQUESTED** [review]
@nelsonrenan very nice translation! :) I opted to keep the name "Dandelion" without translation, as it is a software name. Besides that, I proposed only small changes in your text, I hope you agree with them.

File: monero-outreach-docs/translations/pt-br/archive/dandelion_pt-br.md
Comment (nelsonrenan) [reply monero-outreach-docs/translations/pt-br/archive/dandelion_pt-br.md:6]: Perfect! I had thought about it, but I decided to translate and wait for a suggestion in the review.

**Comment by lh1008** [comment]
hey @nelsonrenan, is everything ready or does it need something else?

**Comment by nelsonrenan** [comment]
Hi @lh1008. Yes, is everything ready.

**Review by netrik182 — APPROVED** [review]
First line still has the original title. Besides that, should be good to go.

**Comment by nelsonrenan** [comment]
Hi @netrik182. The first line will remain as in the original. As recommended by @botoserelepe .

**Review by botoserelepe — APPROVED** [review]

**Review by lh1008 — CHANGES_REQUESTED** [review]
Before merging @nelsonrenan could you please squash commits and make only one. :)

**Comment by erciccione** [comment]
@lh1008 It's ok, i can squash the commits as i usually do for the PRs on this repo. I just need an approval.

**Review by lh1008 — APPROVED** [review]
Approved

**Pull request merged**
